R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
I see where you're coming from

On Wed, Sep 7, 2016 at 11:51 AM, Jordan S Read <notifications@github.com>
wrote:

> In R/metab.bookkeep.R
> <https://github.com/GLEON/LakeMetabolizer/pull/121#discussion_r77850625>:
>
> > @@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
> >  		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
> >  		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
> >  		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
> > -		freq <- nobs
> > +		freq <- nobs - 1
>
> I always felt the opposite, that it should be midnight to midnight, so you
> have a true 24 hour delta.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GLEON/LakeMetabolizer/pull/121/files/c75567a84f7677ee3238146a630d9ff014c790da#r77850625>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFXrf9UbUp-QHXd8AtpRwZSHYgmtSBJuks5qnt1tgaJpZM4Jvljp>
> .
>



-- 
Ryan D. Batt
Aquatic Ecologist
Postdoctoral Researcher, Rutgers University
Website: http://limnology.wisc.edu/personnel/batt/